ydb/core/engine/mkql_engine_flat.cpp

```diff
@@ -798,7 +798,7 @@ class TEngineFlat : public IEngineFlat {
         return result;
     }
 
-    EResult Validate(TValidationInfo& validationInfo) override {
+    EResult ExtractKeys(TValidationInfo& validationInfo) override {
```

Comment:
In the context of old engine this rename is wrong. This method does not extract keys, it actually validates whether it is possible to execute old minikql transaction, with validation info as output. Key extraction is only a useful side-effect.